app/models/question.rb

```diff
@@ -0,0 +1,45 @@
+class Question < ApplicationRecord
+  include Swagger::Blocks
+
+  belongs_to :assessment
+  belongs_to :format
+  belongs_to :created_by, class_name: 'User'
+  has_many :solutions, inverse_of: :question
```

Comment:
`dependent: :destroy`?  Can you do an audit of the has relations and add this where needed?

Reply:
yep will do, although I think we probably won't be deleting assessments it's still to have them ready to cascade delete now while we're working on the code